.github/workflows/main.yml

```diff
@@ -70,6 +70,35 @@ jobs:
           file: build/*/*.dmg
           tag: ${{ github.ref }}
           file_glob: true
+          
+  appcast:
+    name: Set Appcast
+    needs: build
+    runs-on: macos-latest
+    env:
+      JOB_TYPE: BUILD
+      SIGNATURE_FOR_SIGNING: ${{ secrets.SIGNATURE_FOR_SIGNING }}
+    if: github.event_name == 'release'
+    steps:
+      - uses: actions/checkout@v2
+      
+      - name: Get Tag
+        run: TAG_VER=${GITHUB_REF/refs\/tags\//}
+      
+      - name: Get Information & Sign
+        run: |
+           DATE=$(date -R)
+           SIGNATURE=$(./bin/sign_update -s ${{ secrets.SIGNATURE_FOR_SIGNING }} build/Release/MaciASL-${{ env.TAG_VER }}-RELEASE.dmg)
+           ./bin/generate_appcast -o appcast.xml -s ${{ secrets.SIGNATURE_FOR_SIGNING }} --download-url-prefix https://github.com/acidanthera/MaciASL/releases/download/${{ env.TAG_VER }}/ build/Release || exit 1
```

Comment:
`${{ env.TAG_VER }}` will not be available. It should be `${TAG_VER}` here and above.

Reply:
fixed